core/XH.js

```diff
@@ -100,6 +100,7 @@ class XHClass {
     track(opts)                 {return this.trackService.track(opts)}
     fetch(opts)                 {return this.fetchService.fetch(opts)}
     fetchJson(opts)             {return this.fetchService.fetchJson(opts)}
+    setAppHeaders(headers)      {return this.fetchService.setAppHeaders(headers)}
```

Comment:
Concerned it might not be clear enough what we mean by "headers" here - e.g. what "app" means in this context + HTTP request headers vs. some kind of visual element.

What about making the method `FetchService.setDefaultHeaders()`, and then foregoing the XH alias? Presumably an app would need to call the method at most once, so I don't think there's a strong case for hanging an alias off of XH.  Does the word "default" make it clear enough that these will be added on?

Reply:
+1 on these ideas.   One concern is would we have the situation where an app would want these headers set *before* the app starts it Hoist bootstrap sequence?  Imagine a certain type of header is needed at a client-site get past certain internal proxies to Hoist Central itself?

In that case, we would want to add a static defaultFetchHeaders property to the appSpec itself.